modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountChannelDefaults.testcase

```diff
@@ -959,6 +1020,213 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-9704. We can see the price list price that has the highest priority in hierarchy mode(default mode)."
+	@priority = "3"
+	test CanSeeHighestPriorityPriceListPriceInHierarchyMode {
+		property portal.acceptance = "false";
+
+		task ("Given Test Price List 1 with priority 0") {
+			CommerceJSONPriceListsAndPromotionsAPI._addPriceList(
+				catalogName = "Minium",
+				currencyCode = "USD",
+				priceListName = "Test Price List 1");
+		}
+
+		task ("And Test Price List 2 with priority 1") {
+			CommerceJSONPriceListsAndPromotionsAPI._addPriceList(
+				catalogName = "Minium",
+				currencyCode = "USD",
+				priceListName = "Test Price List 2",
+				priority = "1");
+		}
+
+		task ("And Test Price List 1 entry for U-Joint with price set as 20.00 ") {
+			CommercePriceLists.openPriceListsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Test Price List 1");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Entries");
+
+			CommerceJSONPriceListsAndPromotionsAPI._addPriceListEntry(
+				entryPrice = "20",
+				priceListName = "Test Price List 1",
+				productSKU = "MIN55861");
+
+			Refresh();
+
+			Click(locator1 = "Icon#BACK");
+		}
+
+		task ("And Test Price List 2 entry for U-Joint with price set as 18.00 ") {
+			CommerceNavigator.gotoEntry(entryName = "Test Price List 2");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Entries");
+
+			CommerceJSONPriceListsAndPromotionsAPI._addPriceListEntry(
+				entryPrice = "18",
+				priceListName = "Test Price List 2",
+				productSKU = "MIN55861");
+		}
```

Comment:
Navigation can be removed if there's no need to assert the data created by the API call